boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,161 @@
+  of software functionality
+- Expansion options based on Arduino UNO and Pmod™, plus additional expansion 
+  port pins
+- On-board 1.8 V and 3.3 V regulators plus external power supply option
+- 8 Mb Macronix MX25R SPI flash
+- Built-in MCU power consumption and supply voltage measurement
+- UART, I²C and SPI port bridging from LPC54114 target to USB via the on-board 
+  debug probe
+- FTDI UART connector
+
+For more information about the LPC54114 SoC and LPCXPRESSO54114 board:
+
+- `LPC54114 SoC Website`_
+- `LPC54114 Datasheet`_
+- `LPC54114 Reference Manual`_
+- `LPCXPRESSO54114 Website`_
+- `LPCXPRESSO54114 User Guide`_
+- `LPCXPRESSO54114 Schematics`_
+
+Supported Features
+==================
+
+The lpcxpresso54114 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/lpcxpresso54114/lpcxpresso54114_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC54114 SoC has IOCON registers which can be used to configure the
+functionality of a pin. Following pins are configured to support the
+basic blinky and hello world sample applications. 
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_0  | USART           | USART RX                   |
++---------+-----------------+----------------------------+
+| PIO0_1  | USART           | USART TX                   |
++---------+-----------------+----------------------------+
+| PIO0_29 | GPIO            | RED LED                    |
++---------+-----------------+----------------------------+
+| PIO1_10 | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The LPC54114 SoC is configured to use internal FRO at 48MHz as a source for 
+the system clock. Many other sources for system clock is provided in soc which 
```

Comment:
change to:

     Other sources for the system clock are provided in the SOC,
     depending on your system requirements.